frontend/src/views/ParticipantList.js

```diff
@@ -71,12 +78,25 @@ const ParticipantList = observer(() => {
       <PrevPointHeading className={classes.participantsListHeading}>
         Participants
       </PrevPointHeading>
-      <div className="participants">
+      <div className={classes.participants}>
+        <PrevPointPagination
+          participants={participantStore.participants}
+          page={participantTablePage}
+          setPage={setParticipantTablePage}
+        />
         <PrevPointTable
           headerTitles={PARTICIPANT_LIST_TABLE_TITLES}
-          participants={participantStore.participants}
+          participants={participantStore.participants.slice(
+            50 * participantTablePage,
+            50 * participantTablePage + 49
```

Comment:
I pulled down to check this and I was wrong about the 49 value, it skips a person.

 Please change this to 50. sorry.